connectors/sources/dropbox.py

```diff
@@ -189,6 +189,9 @@ async def _set_access_token(self):
         if self.token_expiration_time and (
             not isinstance(self.token_expiration_time, datetime)
         ):
+            self._logger.debug(
+                f"Token expiration time '{self.token_expiration_time}' is not in the correct format. Converting it into ISO format"
+            )
```

Comment:
I don't know that this log provides much value. It isn't a significant code fork path and it isn't changing data being synced, it's just fixing a var type. I think it can be removed.